mteb/tasks/Classification/__init__.py

```diff
@@ -46,6 +46,8 @@
 from .kur.KurdishSentimentClassification import *
 from .mal.MalayalamNewsClassification import *
 from .mkd.MacedonianTweetSentimentClassification import *
+from .mlt.MalteseSentimentClassification import *
```

Comment:
Can you please remove this line? This task doesn't exists anymore, that's why the CI is failing
```suggestion
```